content/docs/command-reference/commit.md

```diff
@@ -140,83 +138,83 @@ stage with `dvc run --no-commit`, or reproduce an entire pipeline using
 development of the stage is finished, `dvc commit` can be used to store data
 files in the cache.
 
-In the `featurize.dvc` stage, `src/featurize.py` is executed. A useful change to
-make is adjusting a parameter to `CountVectorizer` in that script. Namely,
-adjusting the `max_features` value in the line below changes the resulting
-model:
+In the `featurize` stage, `src/featurization.py` is executed. A useful change to
+make is adjusting the `max_features` parameter to `CountVectorizer` in that
+script. The parameters are defined in `params.yaml` file. Updating the value of
+`max_features` to 6000 in `params.yaml` changes the resulting model:
 
-```python
-bag_of_words = CountVectorizer(stop_words='english',
-            max_features=6000, ngram_range=(1, 2))
+```
+featurize:
+  max_features: 6000
+  ngrams: 2
 ```
 
-This edit introduces a change that would cause the `featurize.dvc`, `train.dvc`
-and `evaluate.dvc` stages to execute if we ran `dvc repro`. But if we want to
-try several values for `max_features` and save only the best result to the
-cache, we can run it like this:
+This edit introduces a change that would cause the `featurize`, `train` and
+`evaluate` stages to execute if we ran `dvc repro`. But if we want to try
+several values for `max_features` and save only the best result to the cache, we
+can run it like this:
 
 ```dvc
-$ dvc repro --no-commit evaluate.dvc
+$ dvc repro --no-commit evaluate
 ```
 
-We can run this command as many times as we like, editing `featurize.py` any way
-we like, and so long as we use `--no-commit`, the data does not get saved to the
-cache. Let's verify that's the case:
+We can run this command as many times as we like, editing `featurization.py` any
+way we like, and so long as we use `--no-commit`, the data does not get saved to
+the cache. Let's verify that's the case:
 
 First verification:
 
 ```dvc
 $ dvc status
-
-evaluate.dvc:
-    changed deps:
-        modified:           data/features
-        modified:           model.pkl
-train.dvc:
-    changed outs:
-        not in cache:       model.pkl
+featurize:
+	changed outs:
+	  not in cache:       data/features
+train:
+	changed outs:
+		not in cache:       model.pkl
 ```
```

Comment:
No changes to `evaluate`'s outputs? The text above states it will have changes.

Reply:
This is still pending.

Reply:
Sorry forgot commenting here. 
This is because both `outs` ->`scores.json` and `prc.json` are not tracked by DVC. `cache: false` in [dvc.yaml](https://github.com/iterative/example-get-started/blob/master/dvc.yaml). Changes will be there but they will probably be tracked by git.

Reply:
What I'm saying is that the text above reads "introduces a change that would cause the `featurize`, `train` and `evaluate` stages to execute if we ran `dvc repro`" however status only shows changes to the first 2, so repro would only execute those. The text is wrong, basically.

Reply:
`repro` runs `featurize` also. After running `repro --no-commit`, hash values in `dvc.yaml` gets changed for `plots` and `metrics`. So text is correct right?

Reply:
No. It doesn't seem correct. Why does it even mention the `evaluate` stage?

Reply:
Continued in https://github.com/iterative/dvc.org/pull/1626#pullrequestreview-465506364. Resolving here.

Reply:
> This is because both outs ->scores.json and prc.json are not tracked by DVC

Oh I see you were right about that. So this change was always correct.